modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/processing/NotAcceptedBlock.scala

```diff
@@ -0,0 +1,43 @@
+package org.tessellation.infrastructure.snapshot.processing
```

Comment:
imo such things should be in the domain

Reply:
This file will be removed (conclusion after the discussion about re-using the validators).